go.mod

```diff
@@ -4,9 +4,9 @@ go 1.21
 
 require (
 	github.com/antihax/optional v1.0.0
-	github.com/aws/aws-sdk-go v1.48.0
+	github.com/aws/aws-sdk-go v1.47.9
 	github.com/billgraziano/dpapi v0.5.0
-	github.com/bradleyjkemp/cupaloy/v2 v2.8.0
+	github.com/bradleyjkemp/cupaloy v2.3.0+incompatible
```

Comment:
Looks like a bad merge? We shouldn't be downgrading these dependencies.

Reply:
I saw this and wasn't sure what went wrong, I have the recent changes from `master` and performed `go mod download` and `go mod vendor`. Not sure what's causing this to downgrade.

Reply:
I go the `go.mod` file fixed, but should `go.sum` exist in the PR at all?

Reply:
> should go.sum exist in the PR at all?

Yes, just make sure you run `go mod tidy`!

Reply:
Got it. Pushed the changes after running `go mod tidy`. Thanks!